test/burn/org/apache/cassandra/concurrent/LongSharedExecutorPoolTest.java

```diff
@@ -106,7 +106,7 @@ private void testPromptnessOfExecution(long intervalNanos, float loadIncrement)
     {
         final int executorCount = 4;
         int threadCount = 8;
-        int maxQueued = 1024;
+        int scale = 1024;
```

Comment:
If I'm right this test is not included in CircleCI results. I have started a run in [ci-cassandra](https://ci-cassandra.apache.org/view/all/job/Cassandra-devbranch-test-burn/44/).